config/s2i/create-containers.sh

```diff
@@ -24,4 +25,6 @@ oc new-app rsync-builder ${REPO_URL_PARAM} ${REPO_REF_PARAM}
 oc new-app ostree-compose-builder ${REPO_URL_PARAM} ${REPO_REF_PARAM}
 oc new-app ostree-image-compose-builder ${REPO_URL_PARAM} ${REPO_REF_PARAM}
 oc new-app singlehost-test-builder ${REPO_URL_PARAM} ${REPO_REF_PARAM}
+oc new-app package-test-builder ${REPO_URL_PARAM} ${REPO_REF_PARAM}
```

Comment:
Probably a rebase error, but don't want this line as package-test is gone in favor of singlehost-test

Reply:
I will change it . However, this shell script should not effect the pipeline job.

Reply:
Agreed, it should not. My only point was I think it was accidentally added in and we don't need it as it wouldn't work if the script was run